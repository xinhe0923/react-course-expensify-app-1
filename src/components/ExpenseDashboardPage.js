import React from "react";
import ExpenseList from "./ExpenseList";
import ExpenseListFilters from "./ExpenseListFilters";
import  ExpensesSummary  from "./ExpensesSummary";
import Header from "./Header";

const ExpenseDashboardPage = () => (
  <div>
    <ExpenseListFilters/>

    <ExpenseList />
    <ExpensesSummary/>
  </div>
);
export default ExpenseDashboardPage;
