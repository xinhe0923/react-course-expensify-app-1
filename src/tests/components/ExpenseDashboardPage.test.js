import React from 'react';
import {shallow} from 'enzyme';
import ExpenseDashboardPage from '../../components/ExpenseDashboardPage';


test('should render expenselist item with empty message',()=>{
    
    const wrapper=shallow(<ExpenseDashboardPage  />)
    expect(wrapper).toMatchSnapshot();

})