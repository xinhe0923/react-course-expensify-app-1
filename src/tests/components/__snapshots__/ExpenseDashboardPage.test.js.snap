// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render expenselist item with empty message 1`] = `
<div>
  <Connect(ExpensesSummary) />
  <Connect(ExpenseListFilters) />
  <Connect(ExpenseList) />
</div>
`;
